documentation/reference/document-v1-api-reference.html

```diff
@@ -1,212 +1,342 @@
-          number of documents can be returned.
-          Response may still contain fewer documents if there are not enough matching documents left
-          to visit in the cluster, or if the visiting times out.
-          This parameter is intended for the case when you have relatively few documents in your cluster
-          and where each GET operation otherwise would only return a handful of documents.
-        </p><p>
-          Note that the maximum value of <em>wantedDocumentCount</em> is bounded
-          by an implementation-specific limit to prevent excessive resource usage.
-          If the cluster has many documents (on the order of tens of millions),
-          there is no need to set this value.
-        </p>
-     </td>
-  </tr><tr>
-    <td><strong>concurrency</strong></td>
-    <td>Positive integer. Sends the given number of visitors in parallel to the backend,
-      improving throughput at the cost of resource usage.
-      <em>Caution:</em> given a concurrency parameter of <em>N</em>,
-      the worst case for memory used while processing the request grows linearly with <em>N</em>.
-      This is because the container currently buffers all response data in memory before sending them to the client,
-      and all sent visitors must complete before the response can be sent. Default is 1.
-    </td>
-  </tr><tr>
-    <td><strong>bucketSpace</strong></td>
-    <td>String - explicitly specifies the bucket space to visit. Document types marked as <code>global</code> exist
-        in a separate <em>bucket space</em> from non-global document types. When visiting a particular document type
-        the correct bucket space is automatically deduced based on the provided type name. When visiting at a root
-        <code>/document/v1/</code> level this information is not available, so only the non-global ("default") bucket
-        space is visited by default. Specify <code>global</code> here to visit global documents instead.
-        Supported values: <code>default</code> (for non-global documents) and <code>global</code>.
-  </tr>
+</tr><tr id="continuation">
+  <th>continuation</th>
+  <td>String</td>
+  <td>When visiting, a continuation token is returned if the result set is chunked.
+    Use the token in the <em>continuation</em> parameter to get the next chunk of documents.</td>
+</tr><tr id="create">
+  <th>create</th>
+  <td>Boolean</td>
+  <td>If set to true, updates will create new document if not existing -
+    see <a href="../writing-to-vespa.html#create-if-nonexistent">create if nonexistent</a>.</td>
+</tr><tr id="fieldset">
+  <th>fieldSet</th>
+  <td>String</td>
+  <td>A <a href="../documents.html#fieldsets">field set string</a>
+    with the set of document fields to return.
+    Default is <code>&lt;visited document type&gt;:[document]</code>, returning all fields.
+  </td>
+</tr><tr id="route">
+  <th>route</th>
+  <td>String</td>
+  <td>The route for document operations. Default value is <code>default</code>.
```

Comment:
Default is actually `default-get` for DocumentGet operations, and something very custom for visitors.